db/migrate/20230106195916_add_rapids_columns_to_occupation.rb

```diff
@@ -0,0 +1,8 @@
+class AddRapidsColumnsToOccupation < ActiveRecord::Migration[7.0]
+  def change
+    add_column :occupations, :rapids_code, :string
+    add_column :occupations, :time_based_hours, :integer
+    add_column :occupations, :hybrid_hours, :string
```

Comment:
@Pcushing @jenlu33 do we need to be storing the hybrid hours as either a range, or two separate fields with a min/max?

Reply:
@littleforest I just changed the hybrid hours to an integer range and I also updated the task to set the range.